core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -117,6 +117,10 @@ object ConfigCommand {
             "Invalid entity config: all configs to be added must be in the format \"key=val\".")
     val props = new Properties
     configsToBeAdded.foreach(pair => props.setProperty(pair(0).trim, pair(1).trim))
+    if (props.containsKey(LogConfig.MessageFormatVersionProp)) {
+      println(s"WARNING: The configuration ${LogConfig.MessageFormatVersionProp}=${props.getProperty(LogConfig.MessageFormatVersionProp)} is specified. " +
+        s"This configuration will be ignored if it is not compatible with the inter broker protocol configuration in the Kafka cluster.")
```

Comment:
Reworded this a bit to the following. See if it's better. Ditto in TopicCommand.

This configuration will be ignored if the value is on a version newer than that specified in inter.broker.protocol.version in the broker.